code/code/misc/talk.cc

```diff
@@ -733,69 +737,78 @@ TBeing *findTellTarget(TBeing *me, const sstring &name, bool visible, bool mobs)
 }
 
 
-// returns DELETE_THIS on death of this
-// triggerSpecOnPerson prevents this from being constant
+TBeing *findAccountAlternate(TBeing *me, const sstring &name, bool visible)
+{
+    TBeing *vict;
+    TDatabase db(DB_SNEEZY);
+    db.query("select p1.name as name from player p1, player p2, account a where p2.name='%s' and a.account_id=p2.account_id and p1.account_id=a.account_id", name);
```

Comment:
select player.name as name from player where account_id = %d and name <> %s

Reply:
more readable AND shorter, double win - can't eliminate the join because we don't have the account ID to hand (it's a tell to someone who isn't logged in)

Reply:
I also dedented it because I gave up on my dreams of four-space indents.